dashboard/app/dsl/content_dsl.rb

```diff
@@ -10,6 +10,8 @@ def type(text) @hash[:type] = text end
   def submittable(text) @hash[:submittable] = text end
   def display_name(text) @hash[:display_name] = text end
   def display_as_unplugged(text) @hash[:display_as_unplugged] = text end
+  def use_large_video_player(text) @hash[:use_large_video_player] = text end
+  def hide_reference_area(text) @hash[:hide_reference_area] = text end
```

Comment:
This seems redundant -- can we just hide the reference area whenever use_large_video_player is true?

Reply:
@davidsbailey I had a similar thought and talked it through with Amanda.  We settled on the conclusion that more flexibility to customize this type of level is better since in the future there are likely scenarios where a Levelbuilder might want to show the reference area even if they are also using the large video player.